lib/shopify-cli/shopifolk.rb

```diff
@@ -50,7 +51,9 @@ def reset
     # a valid google cloud config file with email ending in "@shopify.com"
     #
     def shopifolk?
+      return false unless Feature.enabled?('shopifolk-beta')
```

Comment:
Curious as to why this statement is repeated here! Is this because we also expect people to call `Shopifolk.new.shopifolk?` directly?

Reply:
I would expect people to call `check` but as this method is also public I added this extra safeguard.